src/main/java/io/rcktapp/api/handler/s3/S3Db.java

```diff
@@ -102,6 +109,7 @@ public void bootstrapApi() throws Exception
          // Other core metadata includes: eTag, size, lastModified, storageClass
          table.addColumn(new Column(table, "key", "java.lang.String", false));
          addTable(table);
+         log.info("Added bucket: {}" + bucketName.trim());
```

Comment:
We didn't know what buckets it was seeing. This should help us make sure we have what we expect in here.